bookie/static/js/tests/test_user_stats.js

```
@@ -0,0 +1,56 @@
+// Create a new YUI instance and populate it with the required modules.
+YUI.add('bookie-test-userstats', function (Y) {
+    var ns = Y.namespace('bookie.test.userstats');
+    ns.suite = new Y.Test.Suite('User Stats Tests');
+
+    ns.suite.add(new Y.Test.Case({
+        name: "User Stats Tests",
+
+        setUp: function() {
+        },
+
+        tearDown: function() {
+        },
+
+        testApiExists: function() {
+            Y.Assert.isObject(Y.bookie.Api,
+                              "Should find an object for Api module");
+        },
+
+        test_error_message: function() {
+            var userstats_msg = Y.one('#userstats_msg');
+            var API_CFG = {
+                url: 'http://127.0.0.1:6543/api/v1',
```

so we don't want to make real api calls to real urls here. This is part of the mocking setup we talked about. 

See https://github.com/bookieio/Bookie/blob/develop/bookie/static/js/tests/test_view.js#L298

Notice that it creates a fake response and calls the 'error' callback. In this way then you should not make a real http request out to the service.